dts/bindings/misc/nxp,s32-emios.yaml

```diff
@@ -0,0 +1,95 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  NXP S32 Enhanced Modular IO SubSystem (eMIOS) node for S32 SoCs.
+  eMIOS provides independent unified channels (UCs), some of channels
+  have internal counter that either can be used independently or used
+  as a reference timebase (master bus) for other channels.
+
+compatible: "nxp,s32-emios"
+
+include: [base.yaml]
+
+properties:
+  reg:
+    required: true
+
+  clock-divider:
+    type: int
+    required: true
+    description: |
+      Clock divider value for the global prescaler. Could be in range [1 .. 256]
```

Comment:
Isn't the build assertion in the shim driver checking that this value is between 0 and 255?

Reply:
Right, add check for range [1 256]